### ISSUE TITLE
Archiving Khan Academy courses that will be removed

### ISSUE THREAD

**Issue by user655362020** [learningequality/sushi-chef-khan-academy#25]
The courses mentioned in this [article](https://khanacademy.zendesk.com/hc/en-us/articles/360043801271-Khan-Academy-Courses-to-be-Retired-in-Summer-2020) will be removed by July 15, 2020. Is there a way to archive them, preferably in high quality ?

Relevant Issue  :
https://github.com/learningequality/sushi-chef-khan-academy/issues/7

**Comment by ivanistheone** [comment]
Hi @user655362020 ,

Thanks for pointing us to the KA article about the course removal.

> Is there a way to archive them ...

Yes, you are right that creating an archive is a great idea. Here is a general update on that front with links:

*  We've created **two separate archive channels**, one complete archive of all the retired courses that we had (includes only the resources that were openly licensed), and a separate channel just for the exam prep materials.
  * The **Khan Academy (Retired Courses)** contains the complete archive of the retired courses described in the KA article. This channel is PRIVATE, meaning users cannot see it by default (to avoid confusion with the name — we already have too many Khan Academy channels with the same name, so we don't want to confuse things further). Users who want to access the retired courses can import the channel using the token `jahid-zoros`.
  * The Exam Prep materials were copied into a new channel called **[Khan Academy - Standardized Test Preparation](https://kolibri-catalog-en.learningequality.org/en/learn/#/topics/6616efc8aa604a308c8f5d18b00a1ce3)** which is PUBLIC (everyone can see as available to import). This is the channel that most users were interested in, so this is why it is public, and the channel name makes it clear what is inside so there is no chance of confusion.

These two channels are considered "archived" and final so no more edits from now on --- that is the whole point of an archive, right?

Note the [KA English US curriculum](https://kolibri-catalog-en.learningequality.org/en/learn/#/topics/c9d7f950ab6b5a1199e3d6c10d7f0103) and [KA English CBSE India curriculum](https://kolibri-catalog-hi.learningequality.org/hi-in/learn/#/topics/2fd54ca47a8f59c99fcefaaa3894c19e) channels were updated this week to remove the Partner Content and Exam Prep topics preemptively (to avoid having to update the channels again after July 15th).


Thank you very much for raising this issue. Thanks to your heads-up message we were able to save these archived materials.
